<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="4" passed="3" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-08-08T14:58:07 IST" name="Default suite" finished-at="2024-08-08T14:58:18 IST" duration-ms="11106">
    <groups>
    </groups>
    <test started-at="2024-08-08T14:58:07 IST" name="Default test" finished-at="2024-08-08T14:58:18 IST" duration-ms="11106">
      <class name="api.test.UserTests2">
        <test-method is-config="true" signature="setup()[pri:0, instance:api.test.UserTests2@7cc0cdad]" started-at="2024-08-08T14:58:07 IST" name="setup" finished-at="2024-08-08T14:58:07 IST" duration-ms="873" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testPostUser()[pri:1, instance:api.test.UserTests2@7cc0cdad]" started-at="2024-08-08T14:58:08 IST" name="testPostUser" finished-at="2024-08-08T14:58:11 IST" duration-ms="3693" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testPostUser -->
        <test-method signature="testGetUserByName()[pri:2, instance:api.test.UserTests2@7cc0cdad]" started-at="2024-08-08T14:58:11 IST" name="testGetUserByName" finished-at="2024-08-08T14:58:13 IST" duration-ms="1852" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [200] but found [404]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [200] but found [404]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:1240)
at org.testng.Assert.assertEquals(Assert.java:1274)
at api.test.UserTests2.testGetUserByName(UserTests2.java:64)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:962)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:806)
at org.testng.TestRunner.run(TestRunner.java:601)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:433)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:427)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:387)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testGetUserByName -->
        <test-method signature="testUpdateUserByName()[pri:3, instance:api.test.UserTests2@7cc0cdad]" started-at="2024-08-08T14:58:13 IST" name="testUpdateUserByName" finished-at="2024-08-08T14:58:16 IST" duration-ms="3175" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testUpdateUserByName -->
        <test-method signature="testDeleteUserByName()[pri:4, instance:api.test.UserTests2@7cc0cdad]" started-at="2024-08-08T14:58:16 IST" name="testDeleteUserByName" finished-at="2024-08-08T14:58:18 IST" duration-ms="1446" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testDeleteUserByName -->
      </class> <!-- api.test.UserTests2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
